js/src/chat/chat-tools.scss

```diff
@@ -0,0 +1,152 @@
+.shiny-tool-card {
+  max-height: var(--shiny-tool-card-max-height, 500px);
+
+  .tool-intent {
+    opacity: 0.66;
+    font-style: italic;
+    font-weight: normal;
+  }
+
+  .tool-spacer {
+    margin-inline-start: auto;
+  }
+
+  .tool-icon {
+    --_icon-size: var(--shiny-tool-card-icon-size, 16px);
+    width: var(--_icon-size);
+    height: var(--_icon-size);
+    display: flex;
+    align-items: center;
+    flex: none;
+
+    [class^=spinner] {
+      --bs-spinner-width: var(--_icon-size);
+      --bs-spinner-height: var(--_icon-size);
+      --bs-spinner-border-width: 0.20em;
+      color: var(--shiny-tool-card-spinner-color, var(--bs-primary, #007bc2));
+    }
+  }
+
+  .tool-title-name {
+    font-weight: 600;
+  }
+
+  .card-header,
+  .card-footer {
+    font-size: inherit;
+    font-weight: 400;
+    word-break: break-word;
+  }
+
+  >.card-header {
```

Comment:
Do you prefer no space? I prefer space. I can go either way, but there is some inconsistency with space in the selectors